presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -651,14 +651,25 @@ private void assertRoundTrip(List<Type> writeTypes, List<Type> readTypes, List<L
         private final List<FilterFunction> filterFunctions;
         private final Map<Integer, Integer> filterFunctionInputMapping;
         private final Map<Integer, List<Subfield>> requiredSubfields;
-
-        private OrcReaderSettings(Map<Integer, Map<Subfield, TupleDomainFilter>> columnFilters, List<Integer> expectedFilterOrder, List<FilterFunction> filterFunctions, Map<Integer, Integer> filterFunctionInputMapping, Map<Integer, List<Subfield>> requiredSubfields)
+        private final String expectedErrorMessage;
+        private final boolean isExpectedError;
+
+        private OrcReaderSettings(
+                Map<Integer, Map<Subfield, TupleDomainFilter>> columnFilters,
+                List<Integer> expectedFilterOrder,
+                List<FilterFunction> filterFunctions,
+                Map<Integer, Integer> filterFunctionInputMapping,
+                Map<Integer, List<Subfield>> requiredSubfields,
+                boolean isExpectedError,
```

Comment:
Use `Optional<String>` to replace these merge these two fields into one

Reply:
Okay